repository/Seaside-Zinc-Core.package/ZnZincServerAdaptor.class/instance/convertMultipartEntity.with..st

```diff
@@ -0,0 +1,15 @@
+private
+convertMultipartEntity: anEntity with: aMultiValueDictionary
+	anEntity partsDo: [ :part |
+		| value |
+		value := part hasEntity
+			ifFalse: [ String new ]
```

Comment:
Not sure that the compiler can optimise `String new`. Using a literal might make more sense here.

Reply:
Sounds good for me. Changed it.